### ISSUE TITLE
Failed to resolve a sensor class for type: ZLLLightLevel

### ISSUE THREAD

**Issue by simon-lorenz** [peter-murray/node-hue-api#146]
I've run into an issue calling `api.sensors.getAll()`: 

```
ApiError: Failed to resolve a sensor class for type: ZLLLightLevel
    at Object.buildSensor [as build] ([...]/node_modules/node-hue-api/lib/bridge-model/devices/sensors/index.js:32:11)
    at [...]/node_modules/node-hue-api/lib/api/http/endpoints/sensors.js:107:33
```

The light sensor of my [Hue Motion Sensor](https://www2.meethue.com/en-us/p/hue-motion-sensor/046677473389) has the type "ZLLLightLevel", which [this line](https://github.com/peter-murray/node-hue-api/blob/c5dc69d54596c07327cbc31992483967a839ecbb/lib/bridge-model/devices/sensors/index.js#L29) tries to require, but the correct filename would be "ZLLLightlevel".

**Comment by peter-murray** [comment]
Thanks, will issue the fix for this tomorrow.

I have been working on a complete change to the models and building of these objects in a future release.

**Comment by peter-murray** [comment]
I have released version `3.4.1` which has the fix for this in it.

Please let me know if all is well in that as I do not have a motion sensor to test against.

**Comment by simon-lorenz** [comment]
Thanks for the quick fix, it works now!
